### DISCUSSION TITLE
How to create an IOS app using Mojo 🔥 Watch 👇

### DISCUSSION THREAD

**Discussion by pharaohAi** [modular/modular#1973] [General]
I am not an IOS developer. I wonder if this is possible or not 😚 If you are an IOS developer and know the answer, please share the answer with us.

**Comment by hogepodge** [comment]
@pharaohAi please stop posting multiple requests about the roadmap to our forums and GitHub issues. We appreciate your interest, but asking the same question in multiple forums about the roadmap is disruptive. We ask that you please respect our community guidelines and refrain from what can be considered as trolling behavior. You can read our Code of Conduct here. https://github.com/modularml/mojo/blob/main/CODE_OF_CONDUCT.md

**Reply by pharaohAi** [reply]
Thank you, but I was trying to ask the question in different ways so that the reader can answer in different ways. Sorry for the inconvenience. Goodbye 🤝
